### DISCUSSION TITLE
빌드잇 맵 불러올때 504 Gateway Timeout

### DISCUSSION THREAD

**Discussion by DoLongJu** [naverz/zepeto-studio-kor#129] [Issue Report (Legacy)]
빌드잇에서 만든 맵을 유니티에서 불러오기 위해 패키지매니저의 ZEPETO.Buildit 시리즈를 모두 다운받고 ZEPETO.Buildit.Loader를 실행시켰지만 다음과 같은 오류가 생깁니다

혹시 몰라 빌드잇도 재설치하고 맵도 열어보고 컴퓨터와 유니티또한 재시작 해보았습니다만 같은 오류만 뜹니다

해결방법이 있을까요?

![KakaoTalk_20211223_150329833](https://user-images.githubusercontent.com/96557169/147196282-a6c217bb-64fc-41cb-9f33-9781bf9a0383.png)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 해당 오류가 최신 버전 1.1.5 에서도 지속적으로 나타나는지 확인 부탁드립니다.
